CMakeLists.txt

```diff
@@ -294,8 +300,12 @@ if (TILES)
     endif ()
 
     message(STATUS "Searching for SDL2_image library --")
-    find_package(SDL2_image)
-    if (NOT SDL2_IMAGE_FOUND)
+    if(VCPKG_MANIFEST_MODE)
+        find_package(sdl2-image)
+    else()
+        find_package(SDL2_image)
+    endif()
```

Comment:
Can you explain why there's this seemingly minor yet important difference here?

Reply:
Removed in b8c1586

Reply:
sdl2-image is the name used by vcpkg:
```
The package sdl2-image provides CMake targets:

    find_package(sdl2-image CONFIG REQUIRED)
    target_link_libraries(main PRIVATE SDL2::SDL2_image)
```